Raise exception when trying to remove image

smarlowucf opened the pull request:
CLI binary already is handling exceptions.

smarlowucf committed:
Raise exception when trying to remove image

rjschwei commented on lib/gceimgutils/gceremoveimg.py:
```diff
@@ -148,8 +148,13 @@ def remove_images(self):
+                        error=error
+                    )
+                    self.log.error(msg)
+                    raise GCERemoveImgException(msg) from error
```
This breaks the best effort logic, i.e. if we have a number of images the code will exit on the first failure leaving other images that could potentially be deleted as "left overs". We should track the errors and collect the data and then raise after the loop if there is a problem.

smarlowucf replied:
Ack, I overlooked that as I never see this function being used to delete more than a single image at a time :thinking: . Really the code is missing a `remove_image` method.

smarlowucf committed:
Split out a remove image method

This method raises all exceptions. The remove_images method
continues to work as is in that it only logs http error failures.

smarlowucf commented:
@rjschwei I split out a `remove_image` method. The idea is that this could be called and expect all exceptions to propagate. The `remove_images` method continues to work as it did previously. Thoughts?
